### DISCUSSION TITLE
show home page

### DISCUSSION THREAD

**Discussion by samsuljihad** [ENONGLOSKER/ONE-REPO#8] [Show and tell]
![Screenshot (74)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/b9a0b812-1cd7-4aac-81fe-adef1da3bbe5)
![Screenshot (74)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/b9a0b812-1cd7-4aac-81fe-adef1da3bbe5)

**Comment by samsuljihad** [comment]
![Screenshot (73)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/115c8725-0b5b-4368-944f-2f3657993dfb)

**Comment by samsuljihad** [comment]
![Screenshot (71)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/32ef4744-2413-4de4-acd3-3e7682d1d6f0)
![Screenshot (70)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/6b6595c0-9b4c-494a-9195-9d37d045df0e)

**Reply by ENONGLOSKER** [reply]
icon sosial media jangan terlalu mepet, list skill, jangan teralau jarak, card us diferent jangan terlalu jauh jarak tulisan  dengan lingkaran di atas, lingkarnya taruhkan gambar atau icon serta tombol start project nya border radius di perbaiki

**Comment by samsuljihad** [comment]
![Screenshot (67)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/55579d7b-b01f-4d2d-bd81-1034d41df3b6)
![Screenshot (68)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/2e5a3516-c9de-4f28-ad35-67c2726b5253)
![Screenshot (69)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/4470189c-cac7-4816-857a-9566ec0c5ec6)

**Reply by ENONGLOSKER** [reply]
ganti font developer teama menjadi sora font1 dan jangan taruh shadow di pembungkusnya.

**Comment by samsuljihad** [comment]
![Screenshot (72)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/4e34f5d9-b0e5-499f-95d8-a41ec68258dd)

**Comment by samsuljihad** [comment]
sorry gaes,uploadnya tidak beraturan

**Comment by samsuljihad** [comment]
![Screenshot (76)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/dae1f5f5-dbf0-4a5b-b4ea-70552850337d)
![Screenshot (77)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/96517733-f473-41cd-b6c3-257db34c0a5d)
![Screenshot (78)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/fd9ead4a-ffe8-4057-9558-b656f66e5d35)
![Screenshot (75)](https://github.com/ENONGLOSKER/ONE-REPO/assets/134439417/31b9affc-395b-4b67-9429-4babe7244a41)

**Reply by ENONGLOSKER** [reply]
buat jadi satu-satu ke bawah semuanya untuk developer team, dan kasih white space kiri kanan untuk kontent 20px di mobile dan 80px untu dekstok/ipad

**Comment by ENONGLOSKER** [comment]
> ![Screenshot (74)](https://private-user-images.githubusercontent.com/134439417/338630952-b9a0b812-1cd7-4aac-81fe-adef1da3bbe5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxNDQxMTksIm5iZiI6MTcxODE0MzgxOSwicGF0aCI6Ii8xMzQ0Mzk0MTcvMzM4NjMwOTUyLWI5YTBiODEyLTFjZDctNGFhYy04MWZlLWFkZWYxZGEzYmJlNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMVQyMjEwMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZTBlZDVmMzVkYjUxOGZlMjRkMGQ3NTZkNGY4MzU5MDgyY2UxMDkwYTU5ZDE3ZThlMzJhYmQzNGRlZDg1MDkxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.rilyNYfLY_SNIIdwOKn63io85U9mmc3A-0QD2ayUqik) ![Screenshot (74)](https://private-user-images.githubusercontent.com/134439417/338630952-b9a0b812-1cd7-4aac-81fe-adef1da3bbe5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgxNDQxMTksIm5iZiI6MTcxODE0MzgxOSwicGF0aCI6Ii8xMzQ0Mzk0MTcvMzM4NjMwOTUyLWI5YTBiODEyLTFjZDctNGFhYy04MWZlLWFkZWYxZGEzYmJlNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMVQyMjEwMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZTBlZDVmMzVkYjUxOGZlMjRkMGQ3NTZkNGY4MzU5MDgyY2UxMDkwYTU5ZDE3ZThlMzJhYmQzNGRlZDg1MDkxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.rilyNYfLY_SNIIdwOKn63io85U9mmc3A-0QD2ayUqik)

perbaiki

**Comment by ENONGLOSKER** [comment]
perbaiki 
- white space untuk kontent
- shadow
- padding
- kontent
- font
- responsif
